certify-service/src/main/java/io/mosip/certify/utils/KeyManagerCacheUtil.java

```diff
@@ -0,0 +1,30 @@
+package io.mosip.certify.utils;
+
+import io.mosip.certify.core.exception.CertifyException;
+import io.mosip.kernel.keymanagerservice.dto.AllCertificatesDataResponseDto;
+import io.mosip.kernel.keymanagerservice.dto.CertificateDataResponseDto;
+import io.mosip.kernel.keymanagerservice.service.KeymanagerService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.cache.annotation.Cacheable;
+import org.springframework.stereotype.Component;
+
+import java.util.Optional;
+
+@Slf4j
+@Component("keyManagerCacheUtil")
+public class KeyManagerCacheUtil {
```

Comment:
When we discussed, I thought we are adding getVerificationMethod kind of util method in DIDDocumentUtil class with cacheable instead of getting only the kid, any specific reason for this new util class? Can we not make DIDDocumentUtil as a bean and keep the code simple ?